processing/src/main/java/org/apache/druid/math/expr/ExprEval.java

```diff
@@ -142,6 +142,26 @@ public static void serialize(ByteBuffer buffer, int position, ExpressionType typ
     }
   }
 
+  public static byte[] toBytes(ExpressionType expressionType, NullableTypeStrategy<Object> strategy, Object o)
+  {
+    // convert the array to byte[] form so that we take a hash of the whole array
```

Comment:
"so that we take a hash of the whole array" seems strangely specific for a public static method on `ExprEval`

Reply:
oops, i moved this code here from where it was in hll sketch or.. bloom filter or .. somewhere else originally where the comment made more sense